unittests/delay_tests.cpp

```diff
@@ -1439,7 +1439,7 @@ BOOST_AUTO_TEST_CASE( canceldelay_test ) { try {
 
 // test canceldelay action under different permission levels
 BOOST_AUTO_TEST_CASE( canceldelay_test2 ) { try {
-   validating_tester chain;
+   validating_tester_no_disable_deferrd_trx chain;
```

Comment:
Need a test that shows that deferred trxs are not allowed (do not validate) in a block after activation.

Reply:
A test `disable_deferred_trxs_stage_1_retire_test` was added.